manifest.xml

```diff
@@ -22,11 +22,6 @@
     <SourceLocation DefaultValue="https://localhost:3000/taskpane.html"/>
   </DefaultSettings>
   <Permissions>ReadWriteDocument</Permissions>
-  <Requirements>
-    <Sets DefaultMinVersion="1.1">
-        <Set Name="CustomFunctionsRuntime" MinVersion="1.1"/>
```

Comment:
Why is the requirement set not needed anymore?

Reply:
Is was removed because of bug 3220358. Manifest validation fails with it included. Once that is fixed, we can put it back.

Reply:
It's removed from the TypeScript template already. This just merges the same changes over tot he JavaScript template.